pkg/internal/codegen/parse/util.go

```diff
@@ -144,6 +160,20 @@ func HasSubresource(t *types.Type) bool {
 	return false
 }
 
+// hasReadyStatusTag returns true if t is an APIResource annotated with
+// +kubebuilder:printcolumn:ready
+func hasReadyStatusTag(t *types.Type) bool {
+	if !IsPrintColumn(t) {
+		return false
+	}
+	for _, c := range t.CommentLines {
+		if strings.Contains(c, "+kubebuilder:printcolumn:ready") {
```

Comment:
This is no longer required. Read my comment about `ready` in the parse method.

Reply:
yeah